server/db/migrations/20_create_grid_service_instance.rb

```diff
@@ -1,7 +1,12 @@
 class CreateGridServiceInstance < Mongodb::Migration
   def self.up
     GridServiceInstance.create_indexes
-    Container.unscoped.where(container_type: 'container').includes(:grid_service).each do |c|
+    migrate_containers('volume')
+    migrate_containers('container')
```

Comment:
I don't really understand the ordering here, because I'm not sure how the `volume` vs `container` types behaves in regards to having multiple `Container` objects for the same service instance on different host nodes... is there some specific reason to migrate both `volume` and `container` types separately, and in this specific order? As opposed to just `Container.unscoped.includes(:grid_service).each`?